Docs/source/developers/checksum.rst

```diff
@@ -61,3 +61,21 @@ Since this will automatically change the JSON file stored on the repo, make a se
 
    git add <test name>.json
    git commit -m "reset benchmark for <test name> because ..." --author="Tools <warpx@lbl.gov>"
+
+Automated reset of a list of test benchmarks
+--------------------------------------------
+
+You can set the environment variable ``export CHECKSUM_RESET=ON`` before running the ``run_test.sh`` script to reset a series of tests.
+
+Note that in CI, these tests are also additionally run with ``export OMP_NUM_THREADS=1``.
```

Comment:
This seems to imply that all CI tests are run with a single thread, is it really the case? (is the `numthreads` parameter in `WarpX-tests.ini` ignored?)